syncserver/src/server/user_agent.rs

```diff
@@ -1,3 +1,7 @@
+use std::fmt;
+use std::str::FromStr;
+
+use actix_http::header::ACCESS_CONTROL_REQUEST_HEADERS;
```

Comment:
I don't think we need this, since we don't reference it (kinda surprised that it didn't get flagged by `cargo`
```suggestion
```